app/controller/button/SpatialQueryButton.js

```diff
@@ -12,6 +12,24 @@ Ext.define('CpsiMapview.controller.button.SpatialQueryButtonController', {
      */
     drawQueryInteraction: null,
 
+    /**
+     * The {ol.interaction.Modify} used to modify the geometry used in the spatial
+     * query
+     */
+    modifiyQueryInteraction: null,
+
+    /**
+     * The {ol.interaction.Snap} used to snap to the points of the geometry used
+     * in the spatial query
```

Comment:
Missing `@param` declaration

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/188/commits/23701a7cc3461e38fa15bb76ca3721c298b8f3d2